New package: GattinoPleths v0.0.9

JuliaRegistrator opened the pull request:
- Registering package: GattinoPleths
- Repository: https://github.com/ChifiSource/GattinoPleths.jl
- Created by: @emmaccode
- Version: v0.0.9
- Commit: f7e8b46ca29922f110673bf8ef1696ce116902e1
- Reviewed by: @emmaccode
- Reference: https://github.com/ChifiSource/GattinoPleths.jl/issues/2#issuecomment-2492981851
- Description: high-level chorepleth visualizations for gattino
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
#### pleths
This is a pure version of `GattinoPleths` with the MVP of coloring the pleths with a legend. Future iterations will expand on this, providing more options for legends and eventually alongside `GattinoInteractive`, *interactivity*.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598e625b938882bfc7a84860074fb6cfa5f373367fc428fd95a486c9c1bf9b73be145d68ae0bf403aedf56344f17ea669eac02bb9fa58d433218477bc83c768ead5313d24d4bae0be1e0e9956540622c33e805fc14fec4518a8cae1b1a651df4efa2f7dfb3d46f22a03eb461c6511c5ba04916d4a84846b848efecfc2344522ab0e -->

goerz commented:
[noblock] I would recommend setting up tests (the standard `tests/runtests.jl`), and ideally also the continuous integration with Github Actions to run these tests automatically. Even a basic test should help with identifying and fixing the problem that is causing 

> I was not able to load the package (i.e. `import GattinoPleths` failed).

JuliaRegistrator committed:
New package: GattinoPleths v0.0.9

UUID: 9f033946-9ab4-431a-b842-48f7db4940ab
Repo: https://github.com/ChifiSource/GattinoPleths.jl.git
Tree: 28840fb57ec84145db04203aaf769184df2a04b1

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
